### DISCUSSION TITLE
Text Size

### DISCUSSION THREAD

**Discussion by rdixitv** [AppFlowy-IO/AppFlowy#3711] [General help wanted]
Is there a way to change the text size globally? I find the default size very small. I did see that you can change it with the hamburger menu in a text-based note, but this only applies to the size there. It's still small in other types of notes like boards.

Thanks for the help.

**Comment by Twilightssuperb** [comment]
Yep. AppFlowy needs UI zooming so much. For different kind of PC screen resolutions. There are still some opened issues about it, #128 e.g.,and it seems to be added to roadmap as Planned, but...

**Comment by rdixitv** [comment]
Closing as I didn't see that this was a duplicate.
